typescript/infra/scripts/secret-rpc-urls/set-rpc-urls.ts

```diff
@@ -1,22 +1,28 @@
-import { setAndVerifyRpcUrls } from '../../src/utils/rpcUrls.js';
-import { getArgs, withChainRequired, withRpcUrls } from '../agent-utils.js';
+import { setRpcUrlsInteractive } from '../../src/utils/rpcUrls.js';
+import {
+  assertCorrectKubeContext,
+  getArgs,
+  withChains,
+} from '../agent-utils.js';
+import { getEnvironmentConfig } from '../core-utils.js';
 
 async function main() {
-  const { environment, chain, rpcUrls } = await withRpcUrls(
-    withChainRequired(getArgs()),
+  const { environment, chains } = await withChains(getArgs()).alias(
```

Comment:
We could use `withChainsRequired`?

Reply:
`withChainsRequired` doesn't itself exist yet but would be glad if it did!